source/Mlos.Agent.Server/MlosAgentServer.cs

```diff
@@ -90,7 +90,7 @@ public static void Main(string[] args)
                 Console.WriteLine("Connecting to the Mlos.Optimizer");
 
                 // This switch must be set before creating the GrpcChannel/HttpClient.
-                //

```

Comment:
I think generally the preferred style is with the extra blank comment line separator.